code/game/objects/items/weapons/material/twohanded.dm

```diff
@@ -173,10 +173,10 @@
 	force = 10
 	w_class = ITEMSIZE_LARGE
 	slot_flags = SLOT_BACK
-	force_divisor = 0.5 			// 15 when wielded with hardness 30 (glass)
-	unwielded_force_divisor = 0.375
-	thrown_force_divisor = 1.5 		// 22.5 when thrown with weight 15 (glass)
-	throw_speed = 3
+	force_divisor = 0.25 			// 6 when wielded with hardness 30 (glass)
```

Comment:
even though spears are arguably the best melee weapon in the game right now, this nerf is pretty harsh. Consider dropping it to 0.4/0.35?

Reply:
.35 Might be a good spot to reduce.